sbin/create_genesis.sh

```diff
@@ -1,9 +1,49 @@
 #!/bin/bash
-SBIN=`dirname $0`
-SBIN="`cd "$SBIN"; pwd`"
-. $SBIN/configure.sh
+if [ ! -d "$CONFIG_DIR" ]; then
+    SBIN=`dirname $0`
+    SBIN="`cd "$SBIN"; pwd`"
+    . $SBIN/configure.sh
+    CONFIG_DIR="`cd "$CONFIG_DIR"; pwd`"
+fi
+echo "Using $CONFIG_DIR as HH proj"
 
-cd $CONFIG_DIR
+# Check that the dotenv exists.
+ENV=".genesis.env"
+if ! test -f $ENV; then
+    echo "Expected dotenv at $ENV (does not exist)."
+    exit
+fi
+echo "Using dotenv: $ENV"
+. $ENV
 
-npx ts-node src/create_genesis.ts --in data/base_genesis.json --out ../e2e/data/genesis.json
+# Define a function to convert a path to be relative to another directory.
+relpath () {
+    echo `python3 -c "import os.path; print(os.path.relpath('$1', '$2'))"`
```

Comment:
add `python3` dependency in README?

Reply:
we don't currently have any dependencies in the README; i can add this later